### ISSUE TITLE
Scrollable favorites on profile for characters and staff

### ISSUE THREAD

**Issue by psisaipsi** [hohMiyazawa/Automail#283]
It works for anime and manga so it would be really nice to be able to see more than 25 characters and staff on the main profile page.

**Comment by hohMiyazawa** [comment]
Yeah, isn't too difficult. I just put off working on it until somebody asked :)
